#!/usr/bin/env python
PACKAGE = "dynamixel_cpp"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("motor_1", double_t, 0, "Motor 1", 0, -3, 3)
gen.add("motor_2", double_t, 0, "Motor 2", 0, -3, 3)

exit(gen.generate(PACKAGE, "dynamixel_cpp", "DynControl"))
